python_legacy/iceberg/api/types/conversions.py

```diff
@@ -23,6 +23,14 @@
 from .type import TypeID
 
 
+def decimal_to_bytes(type_id, value):
```

Comment:
I see that you added `type_id` since you needed to match the other mappings, but since it's unused it might be helpful to signify that with an underscore. `def decimal_to_bytes(_, value):`.